content/integrate/redis-data-integration/installation/install-k8s.md

```diff
@@ -0,0 +1,472 @@
+
+## Before you install
+
+Complete the following steps before running Helm:
+
+-   [Create the RDI database](#create-the-rdi-database) on your Redis Enterprise cluster.
+-   Create a [user]({{< relref "/operate/rs/security/access-control/create-users" >}})
+    for the RDI database if you prefer not to use the default password (see
+    [Access control]({{< relref "/operate/rs/security/access-control" >}}) for
+    more information).
+-   Download the RDI helm chart tar file from the [download center](https://cloud.redis.io/#rlec-downloads).
+-   If you want to use a private image registry,
+    [prepare it with the RDI images](#using-a-private-image-registry).
+
+### Create the RDI database
+
+RDI uses a database on your Redis Enterprise cluster to store its state
+information. *This requires Redis Enterprise v6.4 or greater*.
+
+- Use the Redis console to create a database with 250MB RAM with one primary and one replica.
+- If you are deploying RDI for a production environment then secure this database with a password
+  and [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security).
+
+You should then provide the details of this database in the [`values.yaml`](#the-valuesyaml-file)
+file as described below.
+
+### Using a private image registry
+
+Add the RDI images from [Docker Hub](https://hub.docker.com/) to your local registry.
+The example below shows how to specify the registry and image pull secret in the
+[`values.yaml`](#the-valuesyaml-file) file for the Helm chart:
+
+```yaml
+global:
+ imagePullSecrets: []
+ # - name: "image-pull-secret"
+ # or
+ # - "image-pull-secret"
+
+
+ image:
+   registry: docker.io
+   repository: redis
+```
+
+To pull images from a local registry, you must provide the image pull secret and in some cases also set the permissions. Follow the links below to learn how to use a private registry with:
+
+-   [Rancher](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-resources-setup/kubernetes-and-docker-registries#using-a-private-registry)
+-   [OpenShift](https://docs.openshift.com/container-platform/4.17/openshift_images/managing_images/using-image-pull-secrets.html)
+-   [Amazon Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/ECR_on_EKS.html)
+-   [Google Kubernetes Engine (GKE)](https://cloud.google.com/artifact-registry/docs/pull-cached-dockerhub-images)
+-   [Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/cluster-container-registry-integration?tabs=azure-cli)
+
+## Install the RDI Helm chart
+
+1.  Decompress the tar file:
+
+    ```bash
+	tar -xvf rdi-k8s-<rdi-tag>.tar.gz 
+    ```
+
+1.  Open the `values.yaml` file and set the appropriate values for your installation
+    (see [The `values.yaml` file](#the-valuesyaml-file) below for the full set of
+    available values).
+
+1.  Start the installation:
+
+    ```bash
+    helm install <The logical chart name> rdi-k8s/<rdi-tag>/helm 
```

Comment:
please add --create-namespace -n rdi at the end of the helm install command

Reply:
Done.